android/app-newm/src/main/java/io/newm/screens/library/NFTLibraryPresenter.kt

```diff
@@ -15,27 +16,38 @@ import io.newm.feature.musicplayer.rememberMediaPlayer
 import io.newm.feature.musicplayer.service.MusicPlayer
 import io.newm.shared.public.models.NFTTrack
 import io.newm.shared.public.usecases.ConnectWalletUseCase
+import io.newm.shared.public.usecases.HasWalletConnectionsUseCase
 import io.newm.shared.public.usecases.WalletNFTTracksUseCase
 import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
 
 class NFTLibraryPresenter(
     private val navigator: Navigator,
+    private val hasWalletConnectionsUseCase: HasWalletConnectionsUseCase,
     private val connectWalletUseCase: ConnectWalletUseCase,
-    private val walletNFTTracksUseCase: WalletNFTTracksUseCase
+    private val walletNFTTracksUseCase: WalletNFTTracksUseCase,
+    private val scope: CoroutineScope,
```

Comment:
The idea is that we want the call to only be made once per app launch. Since this is a top level tab, I don't think this will ever get discarded, given there is only a handful of screens.